README.md

```diff
@@ -17,8 +17,8 @@ Example video: http://www.youtube.com/watch?v=UuTL3nj9fIE
 
 Simply reference TickSpec via [NuGet or Paket](https://www.nuget.org/packages/TickSpec/), download the assembly or build the project from source.
 
-- The binary should work cleanly on any .NET Standard 2.0, .NET 4.5 or later environment.
-- The TickSpec solution file works with Visual Studio 2017. 
+- The binary should work cleanly on any .NET Standard 2.0, .NET 4.7.2 or later environment.
+- The TickSpec solution file works with Visual Studio 2017 and newer (tested with VS 2022).
```

Comment:
Just change this to Visual Studio 2022, we don't need to list all supported versions. Especially when we are not sure whether it still works in the old one (I guess not).

Reply:
Agree https://devblogs.microsoft.com/visualstudio/support-ends-for-older-versions-of-visual-studio-feb2022/

Reply:
I have changed the VS version only.